Orange/widgets/utils/colorgradientselection.py

```diff
@@ -128,12 +129,13 @@ def thresholdHigh(self) -> float:
     def setThresholdHigh(self, high: float) -> None:
         self.setThresholds(min(self.__threshold_low, high), high)
 
-    def center(self) -> float:
+    def center(self) -> Union[float, bool]:
         return self.__center
 
-    def setCenter(self, center: float) -> None:
+    def setCenter(self, center: Union[float, bool]) -> None:
```

Comment:
Optional[float]?

Also 
`centerChanged = Signal(float)` needs to be changed to `centerChanged = Signal(object)`

Reply:
Not really; `Optional[float]` would be equivalent to `Union[float, None]`, while this was really `Union[float, bool]`.

Terrible design. Now constructor gets either a `float` or `None`, in which there is no centering, and can't be re-enable by changing `center`.